f5/bigip/resource.py

```diff
@@ -369,6 +377,48 @@ def raw(self):
         """
         return self.__dict__
 
+    def _key_dot_replace(self, rdict):
```

Comment:
This is required as having dotted keys will prevent you from using this dictionary with dot notation